### PR TITLE
touched up the list~

### PR DESCRIPTION
**Pull request by YukaLily**

### CODE REVIEW

**Comment by YukaLily** [comment]
the final one ;w;

File: json/Item_Stack_Voice.txt [hunk 1]
```diff
@@ -4419,28 +4419,28 @@
 	{
 		"assign": "3180887",
 		"jp_text": "男性［ＥＸ］ボイスＥ０２",
-		"tr_text": "",
+		"tr_text": "Male [EX] Voice E02",
```
Comment (alama) [inline json/Item_Stack_Voice.txt]: `男性Ｃ［ＥＸ］ボイスＥ０２` is already `Male [EX] Voice E02`

File: json/Item_Stack_Voice.txt [hunk 2]
```diff
@@ -5259,203 +5259,203 @@
...
-		"tr_text": "Female Voice Apprentice",
+		"tr_text": "Female Apprentice Voice (All Races)",
 		"jp_explain": "使用すると、新しいボイスが選択可能。\n女性のみ使用可能。\nＣＶ桑島 法子",
 		"tr_explain": "When you use this ticket,\n[Apprentice] can be selected.\nCV: Houko Kuwashima"
 	},
 	{
 		"assign": "31801384",
 		"jp_text": "女性［ＥＸ］ボイスＡ２１",
-		"tr_text": "",
+		"tr_text": "Female [EX] Voice A21",
```
Comment (alama) [inline json/Item_Stack_Voice.txt]: `女性Ｃ［ＥＸ］ボイスＡ２１` is already Female [EX] Voice A21

File: json/Item_Stack_Voice.txt [hunk 3]
```diff
@@ -5560,35 +5560,35 @@
 	{
 		"assign": "31801501",
 		"jp_text": "女性Ｃ［ＥＸ］ボイスＡ２０",
-		"tr_text": "",
+		"tr_text": "Female [EX] Voice A20",
```
Comment (alama) [inline json/Item_Stack_Voice.txt]: `女性Ｃ［ＥＸ］ボイスＡ２０` is already `Female [EX] Voice A20`

**Comment by alama** [comment]
And do not close and make a new pull request, you just can add more commits to your branch

**Pull request merged**